src/avtk/clipselector.cxx

```diff
@@ -247,10 +334,12 @@ int ClipSelector::handle(int event)
 		{
 			// calculate the clicked clip number
 			int clipHeight = (h / numClips);
-			int clipNum = ( (Fl::event_y() ) - y ) / clipHeight;
+			clipNum = ( (Fl::event_y() ) - y ) / clipHeight;
 			if (clipNum >= numClips)
 				clipNum = numClips -1; // fix for clicking the lowest pixel
 
+			return clipNum;
```

Comment:
this return prevents that mouse events can be handled. why do we need this?

Reply:
I fixed this in my testing branch: https://github.com/georgkrause/openAV-Luppp/commit/9516b79b7b10c0ca6d3feac3d88326d30cf46e9f

seems to work, did no deeper testing.

Reply:
same here, no deep tests but the mouse can launch/stop clips now